Change lib and include directory to not conflict with the official package.
Index: ogre1.8-1.8.1/CMake/Templates/OGRE-PCZ.pc.in
===================================================================
--- ogre1.8-1.8.1.orig/CMake/Templates/OGRE-PCZ.pc.in	2012-09-02 07:27:14.000000000 +0200
+++ ogre1.8-1.8.1/CMake/Templates/OGRE-PCZ.pc.in	2012-09-03 18:39:18.672301718 +0200
@@ -1,7 +1,7 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
 libdir=${prefix}/@OGRE_LIB_DIRECTORY@
-includedir=${prefix}/include
+includedir=${prefix}/include/OGRE1.8
 
 Name: OGRE-PCZ
 Description: Portal-Connected Zone Scene Manager for OGRE
Index: ogre1.8-1.8.1/CMake/Templates/OGRE-Paging.pc.in
===================================================================
--- ogre1.8-1.8.1.orig/CMake/Templates/OGRE-Paging.pc.in	2012-09-02 07:27:14.000000000 +0200
+++ ogre1.8-1.8.1/CMake/Templates/OGRE-Paging.pc.in	2012-09-03 18:39:26.820301476 +0200
@@ -1,7 +1,7 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
 libdir=${prefix}/@OGRE_LIB_DIRECTORY@
-includedir=${prefix}/include
+includedir=${prefix}/include/OGRE1.8
 
 Name: OGRE-Paging
 Description: General purpose paging component for OGRE
Index: ogre1.8-1.8.1/CMake/Templates/OGRE-Property.pc.in
===================================================================
--- ogre1.8-1.8.1.orig/CMake/Templates/OGRE-Property.pc.in	2012-09-02 07:27:14.000000000 +0200
+++ ogre1.8-1.8.1/CMake/Templates/OGRE-Property.pc.in	2012-09-03 18:39:30.600301366 +0200
@@ -1,7 +1,7 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
 libdir=${prefix}/@OGRE_LIB_DIRECTORY@
-includedir=${prefix}/include
+includedir=${prefix}/include/OGRE1.8
 
 Name: OGRE-Property
 Description: General purpose property component for OGRE
Index: ogre1.8-1.8.1/CMake/Templates/OGRE-RTShaderSystem.pc.in
===================================================================
--- ogre1.8-1.8.1.orig/CMake/Templates/OGRE-RTShaderSystem.pc.in	2012-09-02 07:27:14.000000000 +0200
+++ ogre1.8-1.8.1/CMake/Templates/OGRE-RTShaderSystem.pc.in	2012-09-03 18:39:34.816301241 +0200
@@ -1,7 +1,7 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
 libdir=${prefix}/@OGRE_LIB_DIRECTORY@
-includedir=${prefix}/include
+includedir=${prefix}/include/OGRE1.8
 
 Name: OGRE-RTShaderSystem
 Description: Real time shader generation component for OGRE
Index: ogre1.8-1.8.1/CMake/Templates/OGRE-Terrain.pc.in
===================================================================
--- ogre1.8-1.8.1.orig/CMake/Templates/OGRE-Terrain.pc.in	2012-09-02 07:27:14.000000000 +0200
+++ ogre1.8-1.8.1/CMake/Templates/OGRE-Terrain.pc.in	2012-09-03 18:40:10.512300177 +0200
@@ -1,7 +1,7 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
 libdir=${prefix}/@OGRE_LIB_DIRECTORY@
-includedir=${prefix}/include
+includedir=${prefix}/include/OGRE1.8
 
 Name: OGRE-Terrain
 Description: Outdoor terrain component for OGRE
Index: ogre1.8-1.8.1/CMake/Templates/OGRE.pc.in
===================================================================
--- ogre1.8-1.8.1.orig/CMake/Templates/OGRE.pc.in	2012-09-02 07:27:14.000000000 +0200
+++ ogre1.8-1.8.1/CMake/Templates/OGRE.pc.in	2012-09-03 18:39:39.468301100 +0200
@@ -1,7 +1,7 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
 libdir=${prefix}/@OGRE_LIB_DIRECTORY@
-includedir=${prefix}/include
+includedir=${prefix}/include/OGRE1.8
 plugindir=${libdir}/OGRE
 
 Name: OGRE
Index: ogre1.8-1.8.1/CMake/Templates/OGREConfig.cmake.in
===================================================================
--- ogre1.8-1.8.1.orig/CMake/Templates/OGREConfig.cmake.in	2012-09-02 07:27:14.000000000 +0200
+++ ogre1.8-1.8.1/CMake/Templates/OGREConfig.cmake.in	2012-09-03 18:39:48.088300846 +0200
@@ -19,8 +19,8 @@
 include(FindPackageMessage)
 
 set(OGRE_PREFIX_DIR "@CMAKE_INSTALL_PREFIX@")
-get_filename_component(OGRE_LIBRARY_DIRS "${OGRE_PREFIX_DIR}/lib" ABSOLUTE)
-get_filename_component(OGRE_INCLUDE_DIRS "${OGRE_PREFIX_DIR}/include/OGRE" ABSOLUTE)
+get_filename_component(OGRE_LIBRARY_DIRS "${OGRE_PREFIX_DIR}/lib/OGRE1.8" ABSOLUTE)
+get_filename_component(OGRE_INCLUDE_DIRS "${OGRE_PREFIX_DIR}/include/OGRE1.8/OGRE" ABSOLUTE)
 set(OGRE_LIBRARIES "OgreMain")
 
 message(STATUS "Found OGRE")
Index: ogre1.8-1.8.1/CMake/Templates/OGREStatic.pc.in
===================================================================
--- ogre1.8-1.8.1.orig/CMake/Templates/OGREStatic.pc.in	2012-09-02 07:27:14.000000000 +0200
+++ ogre1.8-1.8.1/CMake/Templates/OGREStatic.pc.in	2012-09-03 18:39:53.504300686 +0200
@@ -1,7 +1,7 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
 libdir=${prefix}/@OGRE_LIB_DIRECTORY@
-includedir=${prefix}/include
+includedir=${prefix}/include/OGRE1.8
 plugindir=${libdir}/OGRE
 
 Name: OGRE (static lib)
